modules/installation-cluster-quickly-extend-compute-nodes.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc (Installing a cluster on AWS with compute nodes on AWS Local Zones) 
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with compute nodes on AWS Wavelength Zones) 
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:local-zone:
+endif::[]
+ifeval::["{context}" == "installing-aws-wavelength-zone"]
+:wavelength-zone:
+endif::[]
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-cluster-quickly-extend-compute-nodes_{context}"]
+ifdef::local-zone[]
+= Install a cluster quickly in AWS Local Zones
+endif::local-zone[]
+ifdef::wavelength-zone[]
+= Install a cluster quickly in AWS Wavelength Zones
+endif::wavelength-zone[]
```

Comment:
NIT: you are referencing to this section as "`Installing...`" but the title is "`Install..`"

Reply:
I can leave this post-GA as I would need to convert the module to a procedure.